### PR TITLE
Create new_uberon_import.md

### PR DESCRIPTION
**Pull request by lubianat**
First shot at adding documentation for UBERON imports. Sorry if I got anything wrong!

### CODE REVIEW

File: docs/new_uberon_import.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+For updating the imports, the step-by-step is:
+
+- Make sure you are working on a new branch, in the context of the ticket you are working on 
+
+- Add the UBERON term to the end of the list in `cell-ontology/src/ontology/imports/uberon_terms.txt`
+
+- Run `./run.sh make imports/uberon_import.owl`
+
+An  script will wherever necessary. Note that this step might use some memory (at least 2 GB of RAM), so make sure you have some free space before starting it.
```
Comment (matentzn) [inline docs/new_uberon_import.md:13]: Incomplete sentence

File: docs/new_uberon_import.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Often an UBERON term that is not yet imported by CL will be necessary (e.g. for a nice logic definition). 
+
+For updating the imports, the step-by-step is:
+
+- Make sure you are working on a new branch, in the context of the ticket you are working on 
+
+- Add the UBERON term to the end of the list in `cell-ontology/src/ontology/imports/uberon_terms.txt`
+
+- Run `./run.sh make imports/uberon_import.owl`
```
Comment (matentzn) [inline docs/new_uberon_import.md:11]: ./run.sh requires people to have done a chmod +x beforehand which many have not; maybe better sh run.sh.

**Review by matentzn — COMMENTED** [review]
@lubianat What would be great if you could actually review
https://obophenotype.github.io/cell-ontology/odk-workflows/UpdateImports.html
Which is the standard ODK documentation for what you are documenting here and improve that, so users of all ontologies will benefit from what you are doing!

**Review by matentzn — COMMENTED** [review]
@lubianat What would be great if you could actually review
https://obophenotype.github.io/cell-ontology/odk-workflows/UpdateImports.html
Which is the standard ODK documentation for what you are documenting here and improve that, so users of all ontologies will benefit from what you are doing!

**Comment by lubianat** [comment]
@matentzn actually, the core documentation is pretty good, I just had not find it. 

"Standard ODK workflows" --> "Imports management" is perhaps a lot of jargon for those (like me) who are still getting started in the world of ontologies. 

I'll close this PR now, we may think about other solutions in the future.

**Comment by matentzn** [comment]
If you have suggestions on how to make this better, you can make an issue here:

https://github.com/INCATools/ontology-development-kit/issues

We want to make this really perfect, so any feedback is appreciated if you have ideas :)

**Pull request closed without merging**